CONTRIBUTING GUIDELINES.md

```diff
@@ -0,0 +1,106 @@
+-> Reporting a bug
+-> Submitting a fix
+-> Proposing new features
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other standard  method with the owners of this repository before making a change.
+
+<h2 align="center"><b>Downloading Git On Windows</b></h2>
+**1.** Browse to the official Git website:[here](https://git-scm.com/downloads)<br>
+**2.** Click the download link for Windows and allow the download to complete.<br>
+<img src="https://phoenixnap.com/kb/wp-content/uploads/2019/12/download-git-for-windows.png" width=600>
+**3.** Browse to the download location (or use the download shortcut in your browser). Double-click the file to extract and launch the installer.<br>
+<img src="https://phoenixnap.com/kb/wp-content/uploads/2019/12/location-git-windows-download.png" width=600>
+**4.** Allow the app to make changes to your device by clicking Yes on the User Account Control dialog that opens.<br>
+**5.** Review the GNU General Public License,and the installer will ask you for an installation location. Leave the default, unless you have reason to change it.<br>
+**7.** Then installer will install  Git with default settings<br>
+**8.** Then create a GitHub account [here](https://github.com/)<br>
+
+<h2 align="center"><b>Pull Request Process</b></h2> 
+**1.**  Fork [this](https://github.com/smaranjitghose/awesome-portfolio-websites.git) repository.
+**2.**  Clone the forked repository.
+
+```
+git clone --depth 1 https://github.com/<your_user_name>/awesome-portfolio-websites.git
+```
+   <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcT5N0HJ9db7jSvcL4dsDscZQBzqQqqKVs0BnO1OVz26glLWKJRY&usqp=CAU" width="300">
+**3.** Navigate to the project directory.
+
+```
+cd awesome-portfolio-websites
+```
+**4.** Add a reference to the original repository.
+
+```
+git remote add upstream  https://github.com/smaranjitghose/awesome-portfolio-websites.git (url of the repository)
+```
+
+**5.** See latest  changes to the repo using
+
+```
+git remote -v
+```
+
+**6.** Create a new branch.
+
+```
+git checkout -b <your_branch_name>
+```
+
+**7.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
+
+```
+git pull upstream master
+```
+
+**8.** Make changes in the source code after getting yourself assigned to the issue by the mentors/developers of the project.
+
+<p align="center"><img width=35% src="https://media2.giphy.com/media/L1R1tvI9svkIWwpVYr/giphy.gif?cid=ecf05e47pzi2rpig0vc8pjusra8hiai1b91zgiywvbubu9vu&rid=giphy.gif"></p>
+
+**9.** Add and then commit your changes.
+
+```
+git add . or git add --all
+```
+```
+git commit -m "<your_commit_message>"
+```
+
+
+**10.** Push your local branch to the remote repository.
+
+```
+git push -u origin <your_branch_name>
+```
+
+**11.** Create a Pull Request!
+
+**12.** Finally, go to your repository in browser and click on `compare and pull requests`.
```

Comment:
11. and 12. Should be combined like:

 " To create a pull request, click on `compare and pull request`.  Please ensure you compare your feature branch to the desired branch of the repo you are suppose to make a PR to. "